src/snapred/backend/dao/request/FarmFreshIngredients.py

```diff
@@ -50,7 +50,7 @@ def version(self, v: Optional[int]):
     ## the below are not-so-fresh, being fiddly optional parameters with defaults
     convergenceThreshold: float = Config["calibration.diffraction.convergenceThreshold"]
     nBinsAcrossPeakWidth: int = Config["calibration.diffraction.nBinsAcrossPeakWidth"]
-    peakIntensityThreshold: float = Config["calibration.diffraction.peakIntensityThreshold"]
+    peakIntensityThreshold: float = None
```

Comment:
Can you just remove this fully instead of making it NONE? Or is this required for pulling the information from the calibrant sample file?

Reply:
I'm going to make this be `peakIntensityThreshold: Optional[float] = None`. There are cases were FarmFreshIngredients are used, but the peakIntensityThreshold number isn't needed.